backend/index.js

```diff
@@ -29,6 +30,15 @@ const runCrawler = async (crawler) => {
 };
 
 const runCrawlers = async () => {
+  logger.debug('Running migrations');
+  await DBMigrate.getInstance(true, {
+    config: '../db/database.json',
+    cmdOptions: {
+      'migrations-dir': '../db/migrations',
+    },
+  }).up();
+  logger.debug('Migrations completed');
```

Comment:
Let's use `console.log` here

Reply:
below in the code logger is used. also when i'm trying to use console there i'm getting 'ESLint: Unexpected console statement.(no-console)'

Is there any reason to not use logger (it's used in backend/index.js to log execution data)